src/components/select/select-list/select-list-container.style.js

```diff
@@ -1,23 +1,18 @@
 import styled from "styled-components";
 import { baseTheme } from "../../../style/themes";
 
-const overhang = 4;
-
 const StyledSelectListContainer = styled.div`
   background-color: white;
   box-shadow: ${({ theme }) => `${theme.shadows.depth1}`};
   position: absolute;
-  z-index: 1000;
```

Comment:
Do we definitely not need this z-index anymore? We've had a fair few z-index issues recently!